plugin-maven/README.md

```diff
@@ -311,6 +312,19 @@ Groovy-Eclipse formatting errors/warnings lead per default to a build failure. T
 </ktfmt>
 ```
 
+<a name="applying-diktat-to-kotlin-files"></a>
+
+### diktat
+
+[homepage](https://github.com/cqfn/diKTat). [changelog](https://github.com/cqfn/diKTat/releases). [code](https://github.com/diffplug/spotless/blob/main/plugin-maven/src/main/java/com/diffplug/spotless/maven/kotlin/Diktat.java). You can provide configuration path manually as `configPath`.
+
+```xml
+<diktat>
+  <version>0.4.0</version> <!-- optional -->
+  <configPath>"full/path/to/diktat-analysis.yml"</configPath> <!-- optional, configuration file path -->
```

Comment:
These quotes seem like a mistake. Is that correct?

Reply:
Yes. These are version and config file settings, but if they are not specified, the defaults will be used.